nuxeo-ai-core-package/src/main/resources/install/templates/nuxeo-ai-core/config/insight-enrichment-config.xml.nxftl

```diff
@@ -2,20 +2,23 @@
 <component name="org.nuxeo.ai.insight.enrichment.configs">
 
 <#if "${nuxeo.ai.insight.enrichment.enabled}" == "true">
-  <extension point="enrichment" target="org.nuxeo.ai.services.AIComponent">
-    <enrichment name="insight" kind="/prediction/custommodel"
-                class="org.nuxeo.ai.model.serving.InsightProvider" maxSize="10000000">
-    </enrichment>
-  </extension>
-
   <extension target="org.nuxeo.runtime.stream.service" point="streamProcessor">
-      <streamProcessor name="basicProcessor" defaultConcurrency="1" defaultPartitions="4"
+      <streamProcessor name="insightProcessor" defaultConcurrency="1" defaultPartitions="4"
                        class="org.nuxeo.ai.enrichment.EnrichingStreamProcessor"
-                       logConfig="${nuxeo.ai.stream.config.name}">
```

Comment:
So we don't have this variable anymore? just checking

Reply:
it's `logConfig="bulk"` on line 15 there. It was explicitly set to that name because the rest of this particular pipeline uses it. I had a choice between moving everything to the template and a big refactor or hardcode it for these two processors